### PR TITLE
Describe webhook trigger allowed_methods/local_only options

### PR DESCRIPTION
**Pull request by esev**
## Proposed change

Describe the two webhook trigger options added in https://github.com/home-assistant/core/pull/66494 adds two options (_allowed_methods_ & _local_only_).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/66494
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes https://github.com/home-assistant/core/issues/65348

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by dongore — DISMISSED** [review]

**Review by davet2001 — DISMISSED** [review]
The edits look ok to me! Thanks!

File: source/_docs/automation/trigger.markdown [hunk 1]
```diff
@@ -784,7 +788,9 @@ You can run this automation by sending an HTTP POST request to `http://your-home
 curl -X POST -d '{ "key": "value" }' https://your-home-assistant:8123/api/webhook/some_hook_id
 ```
 
-Webhooks support HTTP POST, PUT, and HEAD requests; POST requests are recommended. HTTP GET requests are not supported.
+Webhooks support HTTP POST, PUT, HEAD, and GET requests; PUT requests are recommended. HTTP GET and HEAD requests are not enabled by default but can be enabled by adding them to the _allowed_methods_ option. The request methods can also be configured in the UI by clicking the settings gear menu button beside the Webhook ID.
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]:
```suggestion
Webhooks support HTTP POST, PUT, HEAD, and GET requests; PUT requests are recommended. HTTP GET and HEAD requests are not enabled by default but can be enabled by adding them to the `allowed_methods` option. The request methods can also be configured in the UI by clicking the settings gear menu button beside the Webhook ID.
```

File: source/_docs/automation/trigger.markdown [hunk 2]
```diff
@@ -784,7 +788,9 @@ You can run this automation by sending an HTTP POST request to `http://your-home
...
+
+By default webhook triggers can only be accessed from devices on the same network as Home Assistant or via [Nabu Casa Cloud webhooks](https://www.nabucasa.com/config/webhooks/). The _local_only_ option should be set to _false_ to allow webhooks to be triggered directly via the internet. This option can also be configured in the UI by clicking the settings gear menu button beside the Webhook ID.
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]:
```suggestion
By default, webhook triggers can only be accessed from devices on the same network as Home Assistant or via [Nabu Casa Cloud webhooks](https://www.nabucasa.com/config/webhooks/). The `local_only` option should be set to `false` to allow webhooks to be triggered directly via the internet. This option can also be configured in the UI by clicking the settings gear menu button beside the Webhook ID.
```

File: source/_docs/automation/trigger.markdown [hunk 3]
```diff
@@ -807,6 +813,7 @@ Webhook endpoints don't require authentication, other than knowing a valid webho
 - Do not use webhooks to trigger automations that are destructive, or that can create safety issues. For example, do not use a webhook to unlock a lock, or open a garage door.
 - Treat a webhook ID like a password: use a unique, non-guessable value, and keep it secret.
 - Do not copy-and-paste webhook IDs from public sources, including blueprints. Always create your own.
+- Keep the _local_only_ option enabled for webhooks if access from the internet is not required.
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]:
```suggestion
- Keep the `local_only` option enabled for webhooks if access from the internet is not required.
```

**Review by frenck — APPROVED** [review]
Thanks, @esev 👍 

../Frenck

**Pull request merged**